src/styles/components/_myNypl.scss

```diff
@@ -120,11 +120,13 @@
 }
 
 .MobileMyNypl-Wrapper {
-  display: none;
-  position: absolute;
-  left: 0;
-  width: 100%;
-  z-index: 1000;
+  // display: none;
```

Comment:
If the styles are unnecessary.. we should just remove instead of commenting out

Reply:
Yes, also hoping to put that into a scss refactor PR.